lucene/luke/src/java-module/module-info.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Luke : Lucene toolbox project. */
+@SuppressWarnings({"requires-automatic"})
+module lucene.luke {
```

Comment:
@uschindler @dweiss is this correct?
```suggestion
module org.apache.lucene.luke {
```

Reply:
If it compiles, yes.

Reply:
Sorry misunderstood question. I thought you referred to a "required" clause. To not break users of 9.0, use the names as outputted by the gradlew command above.

Reply:
I think we'll need the same change as this here. 
https://github.com/apache/lucene/commit/20cb6817db45d261bf336102f5842674f377d823

Reply:
I don't think main will merge cleanly - I will not have time for this until evening my time.